service/controller/v23/templates/cloudconfig/format_log_volume.go

```diff
@@ -0,0 +1,14 @@
+package cloudconfig
+
+const FormatVarLogService = `
+[Unit]
+Description=Formats EBS volume for log
+Before=docker.service var-log.mount
```

Comment:
@giantswarm/sre what do you think here guys? it is ok to format and mount the log volume before the `docker.service`. Should it maybe run before anything else? Not sure here.

Reply:
cant we use ignition for formatting the disk?

We should do this in some this in the very early stage of init as many services can already log into `/var/log` and it should be ready before something writes there. Maybe check some recommendation how to st properly mount fo /var/log fir linux os

Reply:
> cant we use ignition for formatting the disk?

I don't think I fully get what you mean. Afaik we are not using Ignition in the tenant machines currently. Vol is currently working on it.

Reply:
We are using ignition in tenants, for aws this is true for long time. KVM and Azure was added recently

Reply:
Looks like we dont have easy integration for adding it via ignition, so lets roll with this for now.

Reply:
It is easy to do with ignition. But all the storage stuff is blocked until CoreOS `1995.5.0` i stable